lms/djangoapps/courseware/tests/test_date_summary.py

```diff
@@ -163,64 +162,58 @@ def test_enabled_block_types(self, course_options, expected_blocks):
         self.setup_course_and_user(**course_options)
         self.assert_block_types(expected_blocks)
 
-    # Specific block type tests
-
-    ## Base DateSummary -- test empty defaults
-
-    def test_date_summary(self):
-        self.setup_course_and_user()
-        block = DateSummary(self.course, self.user)
-        html = '<div class="date-summary-container"><div class="date-summary date-summary-"></div></div>'
-        self.assertHTMLEqual(block.render(), html)
-        self.assertFalse(block.is_enabled)
-
-    ## TodaysDate
-
-    def _today_date_helper(self, expected_display_date):
+    @freeze_time('2015-01-02')
+    def test_todays_date_block(self):
         """
         Helper function to test that today's date block renders correctly
         and displays the correct time, accounting for daylight savings
         """
         self.setup_course_and_user()
-        set_user_preference(self.user, "time_zone", "America/Los_Angeles")
         block = TodaysDate(self.course, self.user)
         self.assertTrue(block.is_enabled)
         self.assertEqual(block.date, datetime.now(utc))
-        self.assertEqual(block.title, 'Today is {date}'.format(date=expected_display_date))
-        self.assertNotIn('date-summary-date', block.render())
-
-    @freeze_time('2015-11-01 08:59:00')
-    def test_todays_date_time_zone_daylight(self):
-        """
-        Test today's date block displays correctly during
-        daylight savings hours
-        """
-        self._today_date_helper('Nov 01, 2015 (01:59 PDT)')
```

Comment:
It's tested in the UI toolkit framework. We can also test here, but it actually happens in the JS.